api/capi/src/nnstreamer-capi-single.c

```diff
@@ -500,12 +454,20 @@ ml_single_create_handle (ml_nnfw_type_e nnfw)
   single_h->nnfw = nnfw;
   single_h->state = IDLE;
   single_h->ignore_output = FALSE;
+  single_h->thread = NULL;
 
   ml_tensors_info_initialize (&single_h->in_info);
   ml_tensors_info_initialize (&single_h->out_info);
   g_mutex_init (&single_h->mutex);
   g_cond_init (&single_h->cond);
 
+  single_h->klass = g_type_class_ref (G_TYPE_TENSOR_FILTER_SINGLE);
```

Comment:
As per documentation, you are right. However, as per [function definition](https://github.com/GNOME/glib/blob/3dec72b946a527f4b1f35262bddd4afb060409b7/gobject/gtype.c#L2921), it can return NULL when the requested class type cannot be found